examples/Exploring-different-models.ipynb

```diff
@@ -0,0 +1,802 @@
+{
```

Comment:
<p>I don't think target encoding needs to be treated like a proper noun. Lower case should be fine. +100 for the brief explanation.</p><p> </p><p>+1 for mentioning a characteristic of the dataset--that it has categorical features only.  That's valuable to readers who are not familiar with the data.</p><p> </p><p>Please add a comma after "For example". In the same sentence, do you mind replacing "it" with "target encoding?" (Restating the noun instead of a pronoun (it, that) is tedious, but very clear.)</p><p> </p><p>Sugg to replace "here" link with the article title and domain, maybe "To learn more about TE, see <a href="https://medium.com/rapids-ai/target-encoding-with-rapids-cuml-do-more-with-your-categorical-data-8c762c79e784" target="_blank">Target Encoding with RAPIDS cuML: Do More with Your Categorical Data</a> at medium.com."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/256/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
@mikemckiernan thanks.